shopify-order-to-sap-order/main.bal

```diff
@@ -0,0 +1,108 @@
+import ballerina/http;
+import ballerina/log;
+
+const SAP_REQUEST_PATH = "/purchaseorder/0001/PurchaseOrder";
```

Comment:
```suggestion
const SAP_REQUEST_PATH = "/purchase-order/0001/purchase/order";
```

Reply:
This not an endpoint designed within our code. It's an SAP endpoint. They mandate using camel-case for `PurchaseOrder` word and failing to do so returns a 404